src/main/java/common/commands/AskWatsonCommand.java

```diff
@@ -0,0 +1,41 @@
+package common.commands;
+
+import common.dto.TextMessage;
+import common.interfaces.GameActionContext;
+import java.io.Serial;
+
+// No need for core imports if just using context
+
+public class AskWatsonCommand extends BaseCommand {
+  @Serial private static final long serialVersionUID = 1L;
+
+  public AskWatsonCommand() {
+    super(true); // Requires case to be started
+  }
+
+  @Override
+  protected void executeCommandLogic(GameActionContext context) {
+    // Get the hint or status message from the context
+    String hintResult = context.askWatsonForHint(getPlayerId());
+
+    // Check if the result indicates Watson wasn't there or unavailable
+    if (hintResult.equals("Dr. Watson is not here to offer a hint.")
+        || hintResult.equals("Dr. Watson is unavailable in this case.")
+        || hintResult.equals("Your location is unknown.")
+        || hintResult.equals("Dr. Watson's location is unknown.")) {
+      // Just send the status message
+      context.sendResponseToPlayer(getPlayerId(), new TextMessage(hintResult, false));
+      return;
+    }
```

Comment:
hard code equals for messages?
so if you change a single comma in the watson class it will break?